### PR TITLE
Initialize fuzzing for CUPS

### PR DESCRIPTION
**Pull request by IslandCow**
Intent is to ease reporting of fuzzing issues to the project maintainer for the Common Unix Printing system, the widely used printing system for all *nix systems.

### CODE REVIEW

File: projects/cups/project.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+homepage: "https://github.com/apple/cups"
+primary_contact: "skau@chromium.org"
+sanitizers:
+  - address
+  - leak
```
Comment (kcc) [inline projects/cups/project.yaml]: I don't think we support "leak" as a separate tool (leaks will be detected by AddressSanitizer)

File: projects/cups/build.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+./buildconf.sh
+./configure
+make clean
+make
```
Comment (kcc) [inline projects/cups/build.sh]: Does this build the fuzz targets and puts them into $OUT? 
Also, is there a seed corpus?

Comment (IslandCow) [reply projects/cups/build.sh]: Not yet.  I'm working on landing the fuzz target upstream.

Comment (IslandCow) [reply projects/cups/build.sh]: Removed leak

Comment (kcc) [reply projects/cups/build.sh]: Ok. Ping us when at least one target is landed. 
The rest looks good. Happy fuzzing!

**Comment by Dor1s** [comment]
@IslandCow do you have any plans on finishing this?

**Comment by jonathanmetzman** [comment]
Spoke to @IslandCow about this. I think we will land this with the fuzz targets living in OSS-Fuzz for now.

**Pull request closed without merging**